bin/apisix

```diff
@@ -525,6 +535,24 @@ local function check_or_version(cur_ver_s, need_ver_s)
     return true
 end
 
+
+local function system_dns_resolver()
+    local file, err = io.open("/etc/resolv.conf")
+    if err then
+        return nil, err
+    else
```

Comment:
we can use a better code style, and I have wrote an issue: https://github.com/apache/incubator-apisix/issues/1180